presto-iceberg/src/main/java/com/facebook/presto/iceberg/optimizer/IcebergEqualityDeleteAsJoin.java

```diff
@@ -344,7 +350,7 @@ private TableScanNode createNewRoot(TableScanNode node, IcebergTableHandle icebe
                             tableName.getSnapshotId(),
                             tableName.getChangelogEndSnapshot()),
                     icebergTableHandle.isSnapshotSpecified(),
-                    icebergTableHandle.getPredicate(),
+                    TupleDomain.all(),
```

Comment:
same comment

Reply:
The same as above, just to show that it is useless, but could not be null for now.